features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -67,7 +67,7 @@
 				<Required />
 				<DisplayOrder>6</DisplayOrder>
 				<SupportedByDefault />
-                <RegEx>^([a-zA-Z0-9_\+\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,10})+$</RegEx>
+                <RegEx>^([a-zA-Z0-9!#$'+\\=^_.{|}~\-&amp;])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,10})+$</RegEx>
```

Comment:
Don't we have to escape +,- and . ?

Reply:
We don't need to escape characters inside [ ]. But adding escape for more safety.
Fixed with https://github.com/wso2/carbon-identity-framework/pull/5568/commits/8a00dce0db28649472b6fd4777f9c02f893110a8